config_samples/zksync_testnet_config_L2.json

```diff
@@ -0,0 +1,43 @@
+{
+    "contracts": {
+        "0x3aad916d98a9a19b56336fb44eb734c7acabf069": "ProxyAdmin",
+        "0xfEF1a26853D2a4133017849f5738eaE92fb8117e": "TransparentUpgradeableProxy",
+        "0xA6f68f512F9C96348f7d5c9192782279a93d1C3B": "ZkSyncBridgeExecutor",
+        "0xaA040F9b7e1f1cEe8704e6D076ffD63410Acfd1E": "OssifiableProxy",
+        "0xFC0FDE38a8051c184202b379330B8b2BB04ef17c": "L2ERC20Bridge",
+        "0x948947F2f05864032Fb340E574D0F13C6BA7bE74": "TransparentUpgradeableProxy",
+        "0xece0c449f94a4869d090153b8419bae1488fe667": "ERC20BridgedUpgradeable"
+    },
+    "explorer_hostname": "zksync2-testnet-explorer.zksync.dev",
+    "github_repo": {
+        "url": "https://github.com/txfusion/lido-l2",
+        "commit": "58fe118e1b1e6a0aa7b327e67041aaaa2fe88dbb",
+        "repo_prefix": "zksync"
+    },
+    "dependencies": {
+        "@openzeppelin/contracts": {
+            "url": "https://github.com/OpenZeppelin/openzeppelin-contracts",
+            "commit": "d4fb3a89f9d0a39c7ee6f2601d33ffbf30085322",
+            "repo_prefix": "contracts"
+        },
+        "@openzeppelin/contracts-upgradeable": {
+            "url": "https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable",
+            "commit": "f6febd79e2a3a17e26969dd0d450c6ebd64bf459",
+            "repo_prefix": "contracts"
+        },
+        "@matterlabs/zksync-contracts": {
+            "0": {
```

Comment:
Seems like config became mode difficult and even good naming for some fields won't let uses to fill in it correctly.
I think that mentioning what _repo_prefix_ is and why should we fill in more then one section of config for zksync dependencies would be better.

Reply:
yeah, will lower the nesting level